### ISSUE TITLE
Error while starting the kernel

### ISSUE THREAD

**Issue by thuytrinh00** [spyder-ide/spyder#22726]
### What steps will reproduce the problem?

I am encountering an issue in Spyder related to spaces in my Windows username (e.g., "Thuy Trinh"). The kernel fails to start, displaying errors indicating that paths with spaces are not recognized. 

Steps to Reproduce:

Install Anaconda in a directory with a username that contains spaces.
Create a conda environment and install Spyder.
Attempt to launch Spyder and start the kernel.

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\Thuy Trinh\anaconda3\Lib\site-packages\jupyter_client\manager.py", line 91, in wrapper
    self._ready.set_result(None)
asyncio.exceptions.InvalidStateError: invalid state

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Thuy Trinh\anaconda3\Lib\site-packages\qtconsole\manager.py", line 27, in poll
    super().poll()
  File "C:\Users\Thuy Trinh\anaconda3\Lib\site-packages\jupyter_client\restarter.py", line 143, in poll
    self.kernel_manager.restart_kernel(now=True, newports=newports)
  File "C:\Users\Thuy Trinh\anaconda3\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thuy Trinh\anaconda3\Lib\asyncio\base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\Thuy Trinh\anaconda3\Lib\site-packages\jupyter_client\manager.py", line 596, in _async_restart_kernel
    await self._async_start_kernel(**self._launch_args)
  File "C:\Users\Thuy Trinh\anaconda3\Lib\site-packages\jupyter_client\manager.py", line 94, in wrapper
    self._ready.set_exception(e)
asyncio.exceptions.InvalidStateError: invalid state
```

## Versions

* Spyder version: 5.5.1  (conda)
* Python version: 3.12.7 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.10
* Operating System: Windows-11-10.0.26100-SP0

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0             :  1.4.0 (OK)
chardet >=2.0.0                  :  4.0.0 (OK)
cloudpickle >=0.5.0              :  3.0.0 (OK)
cookiecutter >=1.6.0             :  2.6.0 (OK)
diff_match_patch >=20181111      :  20200713 (OK)
intervaltree >=3.0.2             :  3.1.0 (OK)
IPython >=8.13.0,<9.0.0,!=8.17.1 :  8.27.0 (OK)
jedi >=0.17.2,<0.20.0            :  0.19.1 (OK)
jellyfish >=0.7                  :  1.0.1 (OK)
jsonschema >=3.2.0               :  4.23.0 (OK)
keyring >=17.0.0                 :  24.3.1 (OK)
nbconvert >=4.0                  :  7.16.4 (OK)
numpydoc >=0.6.0                 :  1.7.0 (OK)
paramiko >=2.4.0                 :  2.8.1 (OK)
parso >=0.7.0,<0.9.0             :  0.8.3 (OK)
pexpect >=4.4.0                  :  4.8.0 (OK)
pickleshare >=0.4                :  0.7.5 (OK)
psutil >=5.3                     :  5.9.0 (OK)
pygments >=2.0                   :  2.15.1 (OK)
pylint >=2.5.0,<3.1              :  2.16.2 (OK)
pylint_venv >=3.0.2              :  3.0.3 (OK)
pyls_spyder >=0.4.0              :  0.4.0 (OK)
pylsp >=1.10.0,<1.11.0           :  1.10.0 (OK)
pylsp_black >=2.0.0,<3.0.0       :  2.0.0 (OK)
qdarkstyle >=3.2.0,<3.3.0        :  3.2.3 (OK)
qstylizer >=0.2.2                :  0.2.2 (OK)
qtawesome >=1.2.1                :  1.3.1 (OK)
qtconsole >=5.5.1,<5.6.0         :  5.5.1 (OK)
qtpy >=2.1.0                     :  2.4.1 (OK)
rtree >=0.9.7                    :  1.0.1 (OK)
setuptools >=49.6.0              :  75.1.0 (OK)
sphinx >=0.6.6                   :  7.3.7 (OK)
spyder_kernels >=2.5.0,<2.6.0    :  2.5.0 (OK)
textdistance >=4.2.0             :  4.2.1 (OK)
three_merge >=0.1.1              :  0.1.1 (OK)
watchdog >=0.10.3                :  4.0.1 (OK)
zmq >=22.1.0                     :  25.1.2 (OK)

# Optional:
cython >=0.21                    :  None (NOK)
matplotlib >=3.0.0               :  3.9.2 (OK)
numpy >=1.7                      :  1.26.4 (OK)
pandas >=1.1.1                   :  2.2.2 (OK)
scipy >=0.17.0                   :  1.13.1 (OK)
sympy >=0.7.3                    :  1.13.2 (OK)
```

**Comment by ccordoba12** [comment]
Hey @thuytrinh00, thanks for reporting. This problem is fixed in our latest Spyder 5 version (5.5.6), which is not available in Anaconda yet. Please update by closing Spyder, opening the Anaconda Prompt and running there these commands:

    conda create -n spyder-env -c conda-forge python=3.12 spyder=5.5.6
    conda activate spyder-env
    spyder

If you need other packages for your work, like Pandas or Matplotlib, you need to run

    conda activate spyder-env
    conda install -c conda-forge pandas matplotlib

Let us know if that solves the problem for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
